### PR TITLE
chore: Sorts test groups in Github Action

### PR DESCRIPTION
**Pull request by lantoli**
## Description

Sorts test groups in Github Action

This PR is a follow-up for a comment by @EspenAlbert on: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2156

Link to any related issue(s): CLOUDP-244924

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR. A migration guide must be created or updated if the new feature will go in a major version.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR. A migration guide must be created or updated.
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contributing guides](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/contributing/README.md)
- [x] I have checked that this change does not generate any credentials and that **they are NOT accidentally logged anywhere**.
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code
- [x] If changes include deprecations or removals, I defined an isolated PR with a relevant title as it will be used in the auto-generated changelog.
- [x] If changes include removal or addition of 3rd party GitHub actions, I updated our internal document. Reach out to the APIx Integration slack channel to get access to the internal document.

### CODE REVIEW

**Review by EspenAlbert — APPROVED** [review]
Fantastic 🎉

File: .github/workflows/acceptance-tests-runner.yml [hunk 1]
```diff
@@ -217,27 +192,52 @@ jobs:
...
+            - 'internal/service/ldapverify/*.go'  
+          network:
+            - 'internal/service/networkcontainer/*.go'
+            - 'internal/service/networkpeering/*.go'
+            - 'internal/service/privateendpointregionalmode/*.go'
+            - 'internal/service/privatelinkendpoint/*.go'
+            - 'internal/service/privatelinkendpointservice/*.go'
+            - 'internal/service/privatelinkendpointservicedatafederationonlinearchive/*.go' 
+          project:
+            - 'internal/service/project/*.go'
```
Comment (EspenAlbert) [inline .github/workflows/acceptance-tests-runner.yml]: [nit] Could we sort these too? 😅

Comment (lantoli) [reply .github/workflows/acceptance-tests-runner.yml:222]: it's sorted, isn't it?

Comment (lantoli) [reply .github/workflows/acceptance-tests-runner.yml:222]: i used the same order for the 3 parts: filter outputs, filter files and jobs

Comment (EspenAlbert) [reply .github/workflows/acceptance-tests-runner.yml]: e.g.,:
```yaml
          project:
            - 'internal/service/accesslistapikey/*.go'
            - 'internal/service/project/*.go'
            - 'internal/service/projectinvitation/*.go'  
            - 'internal/service/projectipaccesslist/*.go'
```

Comment (lantoli) [reply .github/workflows/acceptance-tests-runner.yml]: you mean files and packges inside each test group?  good catch!

Comment (EspenAlbert) [reply .github/workflows/acceptance-tests-runner.yml]: Currently, 
image

Comment (lantoli) [reply .github/workflows/acceptance-tests-runner.yml]: done

**Pull request merged**